pkg/controller/hibernation/hibernation_controller.go

```diff
@@ -222,7 +222,10 @@ func (r *hibernationReconciler) Reconcile(ctx context.Context, request reconcile
 		// set hibernating condition to false for unsupported clouds
 		changed := r.setCDCondition(cd, hivev1.ClusterHibernatingCondition, hivev1.HibernatingReasonUnsupported, msg,
 			corev1.ConditionFalse, cdLog)
-		if changed {
+		rChanged := r.setCDCondition(cd, hivev1.ClusterReadyCondition, hivev1.ReadyReasonRunning, clusterRunningMsg,
+			corev1.ConditionTrue, cdLog)
```

Comment:
Also wondering about this one. I _think_ we talked about this very thing when we did the split that created the Ready condition, and decided that we couldn't assert that the cluster is running just because we don't support hibernation on it. That said, I'm not sure what we _would_ expect the Ready condition -- or the Status.PowerState for that matter -- to be in this case.

...yup, [this comment](https://github.com/openshift/hive/pull/1604#discussion_r754459422) led to [this delta](https://github.com/openshift/hive/pull/1604/commits/ed764520405b1793f95fa3466364987e7c0722ef#diff-ccfc206cfc01a633fb96d40c3560e621640ff2f0f249d6326e6d39b18e1a8756R206) which reverted exactly what you have here.

Can you explain why you added this? If the symptom still exists, we can look at how to fix it elsewhere.

Reply:
I mentioned this in the other comment wrt hibernation controller, but there is a problem with claiming clusters that are not in "Running" status (AFAIR).  I remember seeing a specific piece of code that was depended on on the state of "Running" but I will have to dig it out again. Next week I will refresh my memory and we learn this part again.